### PR TITLE
Add unit test for func VersionedJSON

### PR DESCRIPTION
**Pull request by Hellcatlk**
Signed-off-by: zouyu <zouy.fnst@cn.fujitsu.com>

### CODE REVIEW

File: pkg/kopscodecs/codecs_test.go [hunk 1]
```diff
@@ -76,6 +76,41 @@ func TestToVersionedYaml(t *testing.T) {
...
+			obj: &v1alpha2.Cluster{
+				ObjectMeta: metav1.ObjectMeta{
+					CreationTimestamp: testTimestamp,
+					Name:              "hello",
+				},
+				Spec: v1alpha2.ClusterSpec{
+					KubernetesVersion: "1.2.3",
+				},
+			},
+			expected: heredoc.Doc(`{"kind":"Cluster","apiVersion":"kops.k8s.io/v1alpha2","metadata":{"name":"hello","creationTimestamp":"2017-01-01T00:00:00Z"},"spec":{"kubernetesVersion":"1.2.3"}}`),
```
Comment (olemarkus) [inline pkg/kopscodecs/codecs_test.go:94]: I don't think you need to use `heredoc.Doc` here. In the test above, the json is formatted, while here it is not

File: pkg/kopscodecs/codecs_test.go [hunk 2]
```diff
@@ -76,6 +76,41 @@ func TestToVersionedYaml(t *testing.T) {
...
+	}
+	for _, g := range grid {
+		actualBytes, err := ToVersionedJSON(g.obj)
+		if err != nil {
+			t.Errorf("error from ToVersionedJSON: %v", err)
+			continue
+		}
+		actual := string(actualBytes)
+		actual = strings.TrimSpace(actual)
+		expected := strings.TrimSpace(g.expected)
```
Comment (olemarkus) [inline pkg/kopscodecs/codecs_test.go:105]: Given my comment above, you also do not need to trim spaces (there are no spaces in the string even).

File: pkg/kopscodecs/codecs_test.go (hunk 1 above)
Comment (Hellcatlk) [reply pkg/kopscodecs/codecs_test.go:94]: Thanks, done.

**Pull request merged**